css/styles.css

```diff
@@ -1,5 +1,5 @@
 html {
```

Comment:
- [ ] Most of the fonts stylings on your project do not match the one featured on the Figma template.  Please, make them match so the fonts look exactly as on the given design.

## Explanation

Every single font styling presented in the Figma is required, nothing is optional.  We need to check each element on the Figma and make sure the font's styling matches.

## Hints

Look:

![image](https://user-images.githubusercontent.com/57421823/161899503-07d16c98-7e3d-4f1d-bda3-fb3a10435842.png)

Every font styling is there for you to just copy!  There are just 4 fonts you need to match.

- [ ] OPTIONAL:  On Firefox, the buttons on your popup mobile view overflow their container.  I highly suggest you correct this because some job recruiters are known to love that browser.

![image](https://user-images.githubusercontent.com/57421823/161899695-0e8811ca-60b4-42c3-a5fc-353885cc5034.png)

That way, your popup will look exactly like the Figma 💪